agent/tests/local.rs

```diff
@@ -55,6 +55,8 @@ fn regen_config_with_plugins() -> anyhow::Result<()> {
         poll_interval = "1s"
         flush_interval = "5s"
         no_perf_events = false
+        perf_event_test_path = ""
+        powercap_test_path = ""
```

Comment:
this is a weird behavior, this is because the config regen test is done in a test context with --all-features and in this PR I added new configs only available for test-support feature.